README.md

```diff
@@ -50,7 +59,9 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town.castle.guests << town.residents.delete("Belle")
+
```

Comment:
very close, but we can't use `.` syntax in ruby hashes. It would look like this:

```ruby
town[:castle][:guests] << town[:residents].delete("Belle")
```